src/app.tsx

```diff
@@ -32,58 +21,41 @@ export const useStyles = makeStyles(theme => ({
     flexGrow: 1,
     padding: theme.spacing(3),
   },
+  close: {
+    padding: theme.spacing(0.5),
+  },
 }))
 /**
  * The main entry point of your app. You can start h@cking from here ;)
  */
 export const App: React.FunctionComponent = () => {
   const repo = useRepository()
   const [data, setData] = useState<Image[]>([])
-  const [uploaddata, setUploaddata] = useState<boolean>(false)
-  const [selectedimage, setSelectedimage] = React.useState<SelectedImage>({
-    imgIndex: 0,
-    imgPath: '',
-    imgTitle: '',
-    imgDescription: '',
-    imgAuthor: '',
-    imgAuthorAvatar: '',
-    imgCreationDate: '',
-    imgSize: '',
-    imgDownloadUrl: '',
-  })
+  const classes = useStyles()
+  const [isNotificationShown, ShowNotification] = React.useState<boolean>(false)
+  const [uploaddata, setUploaddata] = React.useState<boolean>(false)
+
+  /** Display the notificationbar about successful upload
+   * @param {boolean} switcher Should the notification shown or not
+   */
+  function DisplayNotification(switcher: boolean) {
+    ShowNotification(switcher)
```

Comment:
Honestly I don't see any value wraping this function with another function. Just pass it down as a prop.

Reply:
done